src/easylink/step.py

```diff
@@ -1481,10 +1519,10 @@ def get_implementation_edges(self, edge: EdgeParams) -> list[EdgeParams]:
             for mapping in mappings:
                 # FIXME [MIC-5771]: Fix ParallelSteps
```

Comment:
Is this still open?

Reply:
[Yes](https://jira.ihme.washington.edu/browse/MIC-5771)